### ISSUE TITLE
Reactive data is not updated in real time

### ISSUE THREAD

**Issue by wsw70** [vuejs/devtools#1471]
### Version
6.0.0-beta.12

### Browser and OS info
Chrome 90, Windows 10

### Steps to reproduce
Inspect a page with reactive data, choose a component to show that data. Change the data in the application.

### What is expected?
The data displayed by vue-devtools should change as the data in the application changes.

That was the behaviour for the Vue2 version of vue-devtools

### What is actually happening?
The data displayed by vue-devtoolsdoes not change. I need to switch to another component and switch back to the one I am interested in to see that the changed data is displayed.

In other words, there is no real-time update of the reactive data.

**Comment by wsw70** [comment]
It seems that https://github.com/vuejs/vue-devtools/issues/1456 is related.

**Comment by Akryum** [comment]
Please provide a runnable reproduction of your issue otherwise I can't fix it.

**Comment by wsw70** [comment]
After reinstalling the whole stack on a new computer the problem seems to have gine. I will close for now and reopen if I see the behaviour again - with a working example.
